source/_integrations/unifiprotect.markdown

```diff
@@ -66,7 +66,11 @@ You will need a local user created in your UniFi OS Console to log in with. Ubiq
     * CONTROLLER PERMISSIONS - Under UniFi Protect, select Administrators. **NOTE**: Other roles may work, but only the default Administrators rule is fully tested.
 1. Click *Add* in at the bottom Right.
 
-**HINT**: A few users have reported that they had to restart their UDMP device after creating the local user for it to work. So if you get some kind of *Error 500* when setting up the Integration, try restart the UDMP.
+<div class='note'>
+
+A few users have reported that they had to restart their UDMP device after creating the local user for it to work. So if you get some kind of *Error 500* when setting up the Integration, try to restart the UDMP.
```

Comment:
What about the UNVR/UNVRPROs?

Reply:
That is a good question. I also do not know how important that is anymore since we have made a lot of changes to the config flow.

Reply:
Since they are all technically NVRs we could say NVR.  It seems like it belongs in troubleshooting though as you suggest.

Reply:
Remove it and added a note about 502s as well since I have been seeing those lately with UFP going down.